### PR TITLE
chore: fix two typos in readme file

### PR DESCRIPTION
**Pull request by zebateira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Note that by setting `root` to true, we ensure that no ancestor configuration is
 
 ## Tests
 
-`$ npm test`
```
Comment (satazor) [inline README.md]: Can you restore back the empty spaces so that they are in two separate lines?

**Review by satazor — CHANGES_REQUESTED** [review]
Nice catches! Thanks

File: README.md (hunk 1 above)
Comment (zebateira) [reply README.md]: ops. will do.

Comment (zebateira) [reply README.md]: will rebase this as well

Comment (zebateira) [reply README.md]: done

**Pull request merged**